values.yaml

```diff
@@ -340,6 +340,68 @@ cluster:
       transactionLogMaximumSizeGB: 50
       # The amount of time in milliseconds a follower node would wait before attempting to verify the last committed entry when the first verification is unsuccessful.
       verificationTimeout: 1500
+    tls:
+      # There are 3 possible security mode configurations
+      # NONE
+      # Unsecured communication will be used for the gRPC client-server communications.
+      # If mode is not specified it will use this configuration by default.
+      # DEFAULT
+      # TLS encryption will be used if the TLS assets for configuring the security communications are found.
+      # TLS
+      # TLS encryption will be enforced for the cluster communication.
+      # TLS assets need to be provided and If not properly configured the server will fail to start!
+      # Most of the possible properties can be seen from the official tomcat connector documentation without
+      # the graphdb.raft.security prefix.
+      # Tomcat docs: https://tomcat.apache.org/tomcat-9.0-doc/config/http.html#SSL_Support_-_Certificate
+      # Official GraphDB docs: https://graphdb.ontotext.com/documentation/10.7/directories-and-config-properties.html#cluster-properties
+      mode: DEFAULT
+      privateKey:
```

Comment:
I still don't know which properties I should use and in what scenarios. 
- Add descriptions
- Can they be grouped somehow?